packages/react/src/createReactQueryHooks.ts

```diff
@@ -60,9 +62,8 @@ export function createReactQueryHooks<
       TOutput
     >,
   ): UseQueryResult<TOutput, TRPCClientError> {
-    const path = pathAndArgs[0];
-    const input = pathAndArgs[1];
-    const cacheKey = [path, input ?? null];
+    const [path, input] = pathAndArgs;
+    const cacheKey = [path, input ?? null, CACHE_KEY_QUERY];
```

Comment:
nice. mamespaced keys to make sure it doesnt clash with other usages of react query